src/main/resources/anniv5Resources/localization/eng/pixiepack/Powerstrings.json

```
@@ -0,0 +1,23 @@
+{
+  "${ModID}:FastHandsPower": {
+    "NAME": "Fast Hands",
+    "DESCRIPTIONS": [
+      "After playing a foreign card, draw ",
```

```suggestion
      "After playing a foreign card, draw #b",
```
Numbers are mostly highlighted blue

---

Fixed.